docs/guidance/map/framework_agreements.md

```diff
@@ -85,44 +85,45 @@ The following guidance describes how to model the different stages of a framewor
 
 ### Invitation to participate in the first stage of a framework agreement procedure
 
-* Create a release and add 'tender' to the `tag` array
+* Create a release and add 'tender' to the `tag` array.
```

Comment:
Please make style-only changes in a separate commit, so that I can review each commit individually and focus my effort on the "substantive" commit(s). Right now, this makes the PR much longer to review because instead of maybe 5 lines of relevant diffs I have dozens of lines of styles changes to skip over.

Reply:
Yes, sorry, I realized that as soon as I made it in the same one.